doc/sphinx/userguide/input-tutorial.md

```diff
@@ -228,22 +228,34 @@ and `activation-energy`.
 ## Input Files Distributed with Cantera
 
 Several reaction mechanism files are included in the Cantera distribution, including
-ones that model natural gas combustion (`gri30.yaml`), high-temperature air
-(`air.yaml`), a hydrogen/oxygen reaction mechanism (`h2o2.yaml`), some pure fluids in
-the liquid-vapor region (`liquidvapor.yaml`), and a few surface reaction mechanisms
-(such as `ptcombust.yaml`, `diamond.yaml`, etc.), among others.
+ones that model natural gas combustion [`gri30.yaml`](../examples/input/gri30),
+high-temperature air (`air.yaml`), a hydrogen/oxygen reaction mechanism
```

Comment:
Minor nit: I think that linking to the files is a good idea, but links to `air.yaml` and `liquidvapor.yaml` are missing.